dshalUtils.c

```diff
@@ -196,3 +198,15 @@ void fill_edid_struct(unsigned char *edidBytes, dsDisplayEDID_t *displayEdidInfo
            parse_cea_block(x, displayEdidInfo);
     }
 }
+bool dsIsValidHandle(intptr_t uHandle)
+{
+    size_t index ;
+    bool retValue = false;
+    for (index = 0; index < dsAUDIOPORT_TYPE_MAX; index++) {
```

Comment:
Can this be generic say for video, audio and FPD - will dsAUDIOPORT_TYPE_MAX same for all?